Pod/Classes/DeviceSize/CGFloat+DeviceSize.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CGFloat+DeviceSize.swift
+//  Swiftilities
+//
+//  Created by Rob Cadwallader on 11/4/16.
+//
+//
+
+import Foundation
+
+public extension CGFloat {
+
+    public func proportional(toDeviceSize size: DeviceSize, axis: Axis) -> CGFloat {
+        let defaultDimension = (axis == .x) ? size.dimensions.width : size.dimensions.height
```

Comment:
I think this way is a bit easier to read. Doesn't require somebody to look back up to the variable declaration to see how it was calculated. Comparing performance between the two is hyper-trivial, but I think this one might be infinitesimally better if not exactly the same. Readability is much more important though; that was by far my main consideration.